_posts/blog/2018-10-31/2018-10-31-why-we-are-integrating-ethereum-into-openbazaar.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: "Why We Are Integrating <span class='post-title-extra'>Ethereum</span> into OpenBazaar"
+layout: post
+date: '2018-10-31 08:30:00 -0600'
+social_title: 'Why We Are Integrating Ethereum into OpenBazaar'
+social_description: 'Why We Are Integrating Ethereum into OpenBazaar'
+---
+
+In a previous article we explained [how we’re integrating Ethereum into OpenBazaar](https://openbazaar.org/blog/Escrow-Smart-Contract-Specification-in-OpenBazaar/). In this article we want to talk about _why_ we're integrating [Ethereum](https://www.ethereum.org/).
+
+#### Bitcoin Beginnings
+
+
+Then, in late 2017, bitcoin transaction fees began to rise. [A lot](https://bitinfocharts.com/comparison/bitcoin-transactionfees.htm). Using bitcoin for payments on OpenBazaar became impractical for months. Since payments in non-bitcoin currencies weren't native on OpenBazaar (they were simply Shapefshifted into bitcoin payments) that meant _all_ payments on OpenBazaar were impractical. Users wanted to use alternative currencies, but we were -- effectively -- forcing them to use bitcoin. Use of the platform ground to a halt. It was obvious something needed to change.
```

Comment:
> Then, in late 2017, bitcoin transaction fees began to rise.

What's before the then? Usually then starts a sentence when it's "first X, then Y." Is there a sentence or paragraph missing here?